src/logservice/logminer/ob_log_miner_br_filter.h

```diff
@@ -143,10 +144,11 @@ class ObLogMinerBRFilter: public ILogMinerBRFilter, public BRFilterThreadPool
 
 private:
   int filter_br_(ObLogMinerBR &br, bool &need_filter);
+  int add_filter_(IBRFilterPlugin *brFilterPlugin, const char *filter_cond);
 
 private:
   bool is_inited_;
-  const char *trans_id_cond_;
+  ObLogMinerOpCond op_cond_;
```

Comment:
Don't put this variable in the class, I think it's enough to put  temporary variable in `ObLogMinerBRFilter ::init`